File: JarusTest.API/Models/Quote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JarusTest.API.Models
{
    public class Quote
    {
        public int QuoteId { get; set; }
        public string QuoteNumber { get; set; }
        public string Status { get; set; }
        public string Applicant { get; set; }
        public DateTime QuoteDate { get; set; }
        public DateTime QuoteEffectiveDate { get; set; }
        public decimal PremiumOptionsBasic { get; set; }
        public decimal PremiumOptionsPreferred { get; set; }
        public decimal PremiumOptionsPremier { get; set; }

    }
}

File: JarusTest-UI/src/app/_models/quote.ts
export interface Quote {
    quoteId: number;
    quoteNumber: string;
    status: string;
    applicant: string;
    quoteDate: Date;
    quoteEffectiveDate: Date;
    premiumOptionsBasic: number;
    premiumOptionsPreferred: number;
    premiumOptionsPremier: number;
}
